src/index.html

```diff
@@ -333,6 +333,22 @@ <h1>4+</h1>
                             <h6>Student Organisations</h6>
                         </div>
                     </div>
+                    <div id="carousel" class="carousel slide d-none d-md-block d-lg-none" data-ride="carousel">
```

Comment:
I hope removing d-none may fix the problem

Reply:
New changes have been pushed to `kcclemo:feature/#229_add_carousel_for_mobile_views`